Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM phusion/passenger-ruby27:1.0.12
```

Comment:
What's the reason for choosing `phusion/passenger-ruby27:1.0.12` as the base image instead of an [official ruby image](https://hub.docker.com/_/ruby) (e.g. `ruby:2.7.2`)?

Reply:
Mostly that I've used it on other projects (cypress) for rails Dockerization. I've never looked into how to install postgres client on the official ruby image, that's the only thing I'd be worried about.